tests/unit/src/scene.cpp

```diff
@@ -1,17 +1,26 @@
 #include <papki/fs_file.hpp>
-#include <ruis/render/null/factory.hpp>
+#include <ruis/render/opengles/renderer.hpp>
+#include <ruis/render/scene/gltf_loader.hxx>
+// #include <ruis/render/null/factory.hpp>
 #include <ruis/render/scene/scene.hpp>
 #include <tst/check.hpp>
 #include <tst/set.hpp>
 
 namespace {
-tst::set set("scene", [](tst::suite& suite) {
+const tst::set set("scene", [](tst::suite& suite) {
 	suite.add("basic_read", []() {
-		ruis::render::null::factory rf;
+		ruis::render::opengles::factory rf;
+		// ruis::render::null::factory rf;
```

Comment:
tests are currently failing on CI due to undefined reference to `opengles::factory` etc. This is because you don't link unit test app to the `ruis-render-opengles` library. But the question here is why do you want to use real Opengl ES renderer for tests? Why did you comment the `null` renderer? I think you should be using `null` renderer here.
II tried to debug it a little bit, reverting use of null-renderer, but then the test fails with uncaught exception:
```
failed: scene basic_read
  uncaught exception:
    std::invalid_argument: read_gltf(): n_skip_bytes < 0
```

I tried to eyeball the problem in the code, but couldn't, it needs deeper investigation.
So far, the code around throwing the exception looks suspicious:
```
        int n_skip_bytes = int(acc_stride) - int(sizeof(tp_type));
	if (n_skip_bytes < 0) {
		throw std::invalid_argument("read_gltf(): n_skip_bytes < 0");
	}
```
Using debugger I could see that `stride` is 0 in the failing case.
The gltf spec says about buffer view stride:
```
Type: integer
Required: No
Minimum: >= 4
Maximum: <= 252
```
I.e. it should never be 0. Perhaps if it is absent you set it to 0? Need to gracefully handle this case.

Anyway, you  need to fix the test app to make CI pass.